elyra/pipeline/component_parser_kfp.py

```diff
@@ -140,28 +138,25 @@ def _read_component_yaml(self, registry_entry: SimpleNamespace) -> Optional[Dict
                            f"location: '{registry_entry.location}' -> {str(e)}")
             return None
 
-    def _get_adjusted_parameter_fields(self,
-                                       component_body: Dict[str, Any],
-                                       io_object_name: str,
-                                       io_object_type: str,
-                                       parameter_type: str) -> str:
+    def _is_pathbased_parameter(self, parameter_name: str, component_body: Dict[str, Any]) -> bool:
```

Comment:
I think this might read better as `_is_path_based_parameter`.  I find "pathbased" a little tough to visually parse.

Reply:
I can change the name post merge. Have one change that needs to go in as well